### ISSUE TITLE
biom-format not found 

### ISSUE THREAD

**Issue by Nazzy87** [picrust/picrust2#314]
Solving environment fails when installing picrust2. I am using mac m1 and I get below error when I try to install picrust2 env using 
conda env create -f picrust2-env.yaml. Any way around pls? thanks

Solving environment: failed

ResolvePackageNotFound: 
  - biom-format[version='>=2.1.10']
  - hmmer[version='>=3.1b2,<=3.2.1']
  - sepp=4.3.10
  - gappa=0.8.0
  - r-castor[version='>=1.7.2']
  - epa-ng=0.3.8

**Comment by gavinmdouglas** [comment]
Hi there,

I can confirm that I can install PICRUSt2 with conda on my M1 Mac, so I don't think this is a Mac OS X problem. Are you using the latest version of conda?


Cheers,

Gavin

**Comment by SamuelAbip** [comment]
i'm having the same problem on my m1 mac, tried with conda/mamba and installing from bioconda/sources.

**Comment by lcamVz** [comment]
Hi Gavin, 

Would you mind sharing your conda version I had picrust2 running on my intel desktop but since I updated my OSX and conda I am having issues. I also have an M1 Mac and am interested on installing it in there. 

Best regard

**Comment by R-Wright-1** [comment]
Hi there,

I just looked into installation on my M1 mac and found that I was getting similar errors initially.

But everything seems to work fine if I first of all run:
`conda config --add subdirs osx-64`

So for reference, everything that I ran to install PICRUSt2 from source: 
```
wget https://github.com/picrust/picrust2/archive/v2.5.2.tar.gz
tar xvzf  v2.5.2.tar.gz
cd picrust2-2.5.2/
conda config --add channels defaults
conda config --add channels bioconda
conda config --add channels conda-forge
conda config --add subdirs osx-64
conda env create -n picrust2-src -f picrust2-env.yaml
conda activate picrust2-src
pip install --editable .
```

When I initially ran `pytest`, it failed with the same error mentioned in #330, but the fix given there works:
`pip install dendropy==4.5.2`

After installing this version of dendropy, `pytest` ran fine. 
Also note that I called my environment picrust2-src, but you don't need that option. 

Robyn
